crawler/build.sbt

```diff
@@ -11,9 +11,12 @@ scalacOptions ++= Seq (
   "-deprecation",
   "-language:postfixOps",
   "-language:implicitConversions",
+  "-language:higherKinds",
```

Comment:
No, it's a leftover. I had some code requiring it, but now it's gone. Will delete this.